lib/modules/manager/custom/regex/readme.md

```diff
@@ -88,15 +88,13 @@ But you don't want to write a regex custom manager rule for _each_ variable.
 Instead you enhance your `Dockerfile` like this:
 
 ```Dockerfile
-ARG IMAGE=node:12@sha256:6e5264cd4cfaefd7174b2bc10c7f9a1c2b99d98d127fc57a802d264da9fb43bd
-FROM ${IMAGE}
```

Comment:
This type of pattern is now handled natively by the `dockerfile` manager so is not a good example for regex manager